spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -240,7 +240,7 @@ for _, strategy in helpers.each_strategy() do
     it("logs to HTTP", function()
       local res = assert(proxy_client:send({
         method = "GET",
-        path = "/status/200",
+        path = "/status/200?test0=test0&test1=test1",
```

Comment:
This is not testing this change correctly. We need to add a plugin config like ones in https://github.com/Kong/kong/pull/8627/files#diff-50dcfdce82665f85d2027363af6956fc788e5b8f6c6b8de907e9bd82e8871501R202 and add the query parameter in the `config.http_endpoint` and make sure it works.